developers/weaviate/quickstart/index.md

```diff
@@ -257,22 +257,22 @@ Generative search sends retrieved data from Weaviate to a large language model,
 ## Recap
 
 Well done! You have:
-- Created your own cloud-based vector database with Weaviate,
-- Populated it with data objects using an inference API,
+- Created your own cloud-based vector database with Weaviate
+- Populated it with data objects using an inference API
```

Comment:
Consider
Used an API to add data objects

Reply:
The previous statement is comprehensive, so there's no need to modify it again.